### PR TITLE
(IMAGES-918) Localise Admin User/Password

### PR DESCRIPTION
**Pull request by jcoconnor**
Issue with French/German localised Admin names for the scheduled Task work.
Fix so that we can pass the correct Username/Password into the scripts.

### CODE REVIEW

File: templates/win/common/files/Autounattend.xslt [hunk 1]
```diff
@@ -84,11 +84,57 @@
     </xsl:copy>
   </xsl:template>
 
-  <!-- Rule to select appropriate logon sequence -->
+  <!-- Rule to select appropriate logon sequence and perform Administrator/Password substitutions. -->
   <xsl:template match='u:unattend/u:settings/u:component[@name="Microsoft-Windows-Shell-Setup"]/u:PackerLogonSequence'>
        <xsl:if test="@ImageProvisioner=$ImageProvisioner">
           <!-- Strip out PackerLogonSequence once selected to present valid Unattend.xml -->
-          <xsl:copy-of select="node()" />
+          <xsl:for-each select="node()">
```
Comment (Iristyle) [inline templates/win/common/files/Autounattend.xslt]: Can't you use a more direct `match` to get the right nodes picked out rather than looping?

Comment (jcoconnor) [reply templates/win/common/files/Autounattend.xslt]: Tx much for the pointers will absorb them tomorrow.
The good news is that with the minor seq fix below, this appears to come up ok for the EN-us case and also fixes the issue I noticed with Win-2019 ([IMAGES-928](https://tickets.puppetlabs.com/browse/IMAGES-928))

**Comment by jcoconnor** [comment]
@Iristyle I've updated the template handling as per your suggestions.
I have one more fix that possibly could be used to remove the <xsl:choose> from the CommandLine handling rule.

**Comment by jcoconnor** [comment]
Work for this is essentially complete.
Pushing through a full set of builds over the weekend to validate the work.

**Comment by jcoconnor** [comment]
Hmm - spoke too soon - authentication issue with French only - will look at on Monday.

**Comment by jcoconnor** [comment]
OK - Issue with French resolved - appears to be building ok now.

**Comment by jcoconnor** [comment]
All images built and ran acceptance tests overnight, so French issue above is sorted.

**Comment by jcoconnor** [comment]
Still need a small fix to the French Post-Clone startup script. Doing......

**Comment by jcoconnor** [comment]
Ok - this is building and running the acceptance suite on the English and French platforms.
So ready to merge pending review.

**Pull request merged**